modules/core/server/services/influx.server.service.js

```diff
@@ -44,12 +46,12 @@ exports.getClient = function(callback) {
  * influxdb (camelCase)
  * @param {Object} fields - key: value pairs will be saved in influxdb as field
  * key: field value
- * @param {number|Date} [fields.time=new Date()] - time of measurement with precision ms
+ * @param {number|Date} [fields.time=new Date()] - time of measurement with precision nanosecond
```

Comment:
We could consider limiting the `fields.time` to `Date`, which would remove the confusion (node-influx takes care of that), but force the precision to be only _ms_ or lower. It is possible to [get nanosecond precision in node via `process.hrtime()`](http://stackoverflow.com/a/18197438) and if needed, it would have to be passed as a timestamp. The question is, would it ever be needed?